src/main/java/roomescape/member/service/MemberService.java

```diff
@@ -0,0 +1,25 @@
+package roomescape.member.service;
+
+import java.util.List;
+import org.springframework.stereotype.Service;
+import roomescape.member.Member;
+import roomescape.member.dao.MemberDao;
+import roomescape.member.dto.response.MemberResponse;
+
+@Service
+public class MemberService {
+
+    private final MemberDao memberDao;
+
+    public MemberService(MemberDao memberDao) {
+        this.memberDao = memberDao;
+    }
+
+    public List<MemberResponse> findAll() {
+
+        List<Member> MemberDaoAll = memberDao.findAll();
+        return MemberDaoAll.stream()
+                .map(MemberResponse::toDto)
+                .toList();
+    }

```

Comment:
유의미한 변수명을 사용하거나 복잡하지 않다면 한줄로 합쳐도 좋을 것 같아요 ㅎㅎ

```suggestion
   public List<MemberResponse> findAll() {
        return memberDao.findAll()
                .stream()
                .map(MemberResponse::toDto)
                .toList();
    }
```

Reply:
완료했습니다!!

[refactor: 변수 추출 삭제하도록 리팩토링](https://github.com/woowacourse/spring-roomescape-member/pull/320/commits/73cf3e632f928d8f2e00b5fc3f1c44a7b1f8b513)